go/vt/discovery/healthcheck.go

```diff
@@ -207,7 +208,9 @@ func (hcc *healthCheckConn) processResponse(ctx context.Context, hc *HealthCheck
 			hc.addrToConns[key] = hcc
 			hc.addEndPointToTargetProtected(hcc.target, endPoint)
 			hc.mu.Unlock()
+			log.Infof("StatsUpdate(New Backend): cell: %v, target: %+v, endpoint: %+v, reparent time: %v", hcc.cell, hcc.target, endPoint, hcc.tabletExternallyReparentedTimestamp)
```

Comment:
We have thousands of tablets, is that gonna spam the logs at startup?

(I mean, during regular operation, that's probably fine, but startup might be too much to log... not sure).